ironfish-cli/src/commands/wallet/assets.ts

```diff
@@ -27,19 +27,24 @@ export class AssetsCommand extends IronfishCommand {
   static flags = {
     ...RemoteFlags,
     ...TableFlags,
+    account: Flags.string({
+      char: 'a',
+      description: 'Name of the account to get assets for',
+    }),
   }
 
   static args = [
     {
       name: 'account',
       required: false,
-      description: 'Name of the account',
+      description: 'Name of the account. DEPRECATED: use --account flag',
     },
   ]
 
   async start(): Promise<void> {
     const { flags, args } = await this.parse(AssetsCommand)
-    const account = args.account as string | undefined
+    // TODO: remove account arg
+    const account = (flags.account ? flags.account : args.account) as string | undefined
```

Comment:
For this to be right, the cast needs to be on args.account rather than the whole expression. It's the same total result, but before it gets cast it's actually `string | any` -> string | undefined.

```suggestion
    const account = (flags.account ? flags.account : args.account as string | undefined)
```